lib/shopify/customer.ex

```diff
@@ -121,4 +205,15 @@ defmodule Shopify.Customer do
       path: "admin/customers/#{customer_id}.json"
     }
   end
+
+  @doc """
+  Updates a single customer metafield
+  """
+  @spec update_customer_metafield(binary, binary) :: Shopify.Operation.t()
```

Comment:
Since this is on the `Customer` module adding `customer` to the function name is redundant. This can just be `update_metafield/2`.

Reply:
Changes made: c29ac7e..7606d80  feature/metafields -> feature/metafields